monkey/monkey_island/cc/services/bootloader.py

```diff
@@ -17,7 +17,7 @@ def parse_bootloader_telem(telem: Dict) -> bool:
             telem['os_version'] = "Unknown OS"
 
         telem_id = BootloaderService.get_mongo_id_for_bootloader_telem(telem)
-        mongo.db.bootloader_telems.update({'_id': telem_id}, telem, upsert=True)
+        mongo.db.bootloader_telems.update({'_id': telem_id}, {'$setOnInsert': telem}, upsert=True)
```

Comment:
What do you think about logging if something already exists?

Reply:
Log would be redundant, because we expect this to happen 100% of the time